src/effects/RealtimeEffectManager.cpp

```diff
@@ -129,7 +129,7 @@ size_t RealtimeEffectManager::Process(bool suspended, Track &track,
    // Can be suspended because of the audio stream being paused or because effects
```

Comment:
So the meaning of the return value has changed; but it is difficult for me to understand what it is now. Number of processed samples? a comment in the prototype would help

Reply:
Exactly.  The comments in header files are updated.

The sum of latencies of the effects must be discarded but the state object is responsible to report the count, only, not do the discarding.